pydantic/main.py

```diff
@@ -271,6 +289,7 @@ class BaseModel(metaclass=ModelMetaclass):
     model_config = ConfigDict()
     __slots__ = '__dict__', '__pydantic_fields_set__', '__pydantic_extra__'
     __pydantic_model_complete__ = False
+    __pydantic_root_model__: typing.ClassVar[bool] = False
```

Comment:
Root model indicator. This could be transformed into enum in the future if we will need another exotic `BaseModel` subclass.

Reply:
Could we move this annotation to live alongside the others (like `__pydantic_post_init__: typing.ClassVar[...` above), and change this to just `__pydantic_root_model__ = False`? I guess ideally we'd do the same with `__pydantic_model_complete__` (though I thought that got renamed to `__pydantic_complete__`.. maybe not yet? or maybe it will be once merged with main)

Reply:
Well "others" are there under the comment "populated by the metaclass, defined here to help IDEs only". This on the other hand is defined here. First, I did exactly what you are suggesting but then I've stumbled upon that comment and noticed `__pydantic_model_complete__` here. As far as I've understood the code, this is the place for things that are defined here and "others" are there just for type checking purposes and are being effectively defined elsewhere (actually in the metaclass).